o2a/mappers/dummy_mapper.py

```diff
@@ -14,16 +14,37 @@
 # limitations under the License.
 """Dummy Mapper that is used as temporary solution while we are implementing the real mappers.
 """
-from typing import Set
+from typing import Set, Tuple, List
+from xml.etree.ElementTree import Element
 
+from o2a.converter.relation import Relation
 from o2a.converter.task import Task
 from o2a.mappers.action_mapper import ActionMapper
+from o2a.utils.dict_utils import remove_key_from_dictionary_copy
 
 
 class DummyMapper(ActionMapper):
-    def to_tasks_and_relations(self):
-        tasks = [Task(task_id=self.name, trigger_rule=self.trigger_rule, template_name="dummy.tpl")]
-        relations = []
+    """Dummy mapper used in place of not-yet-implemented mappers """
+
+    def __init__(self, oozie_node: Element, name: str, dag_name: str, **kwargs):
+        kwargs, job_properties = remove_key_from_dictionary_copy(kwargs, "job_properties", {})
```

Comment:
I do not understand the purpose of this code

Reply:
The problem was that I wanted to keep configuration/job properties obligatory for all mappers - except kill and dummy mappers. Since we should not use = {} in parameter default values (mutable dict) I tried to use = None before and override it with {} inside but mypy complained about the Optional[Dict]. However I found a simpler way of doing it - see e0ff6f1bf016cd3a42d7e3f57dedd40aade8f748

Reply:
It actually fixed even better in e4afbfddee0b8e9bdea5ff11f797a06b523d39a8